app/src/main/java/org/jellyfin/mobile/player/source/MediaSourceResolver.kt

```diff
@@ -33,6 +33,10 @@ class MediaSourceResolver(private val apiClient: ApiClient) {
                 itemId = itemId,
                 data = PlaybackInfoDto(
                     userId = apiClient.userId,
+                    // We need to remove the dashes so that the server can find the correct media source.
+                    // And if we didn't pass the mediaSourceId, our stream indices would silently get ignored.
+                    // https://github.com/jellyfin/jellyfin/blob/9a35fd673203cfaf0098138b2768750f4818b3ab/Jellyfin.Api/Helpers/MediaInfoHelper.cs#L196-L201
+                    mediaSourceId = itemId.toString().replace("-", ""),
```

Comment:
This conversion feels hacky, do we not have a reference to the mediasource so we can use the id from that?

Reply:
It is hacky but sadly not. The id shouldn't be a string anyways but the api model would require changes for this.

Reply:
@crobibero can you help us out here, would it be possible to 1) change the parameter type to a UUID, or 2) allow a UUID with dashes to be auto-converted serverside?

Reply:
Which parameter do you want changed? `MediaSourceInfo`?

Reply:
The `mediaSourceId`, but I don't know whether it supports more than UUIDs.